.github/workflows/lint.yml

```diff
@@ -12,5 +12,27 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Check Line Endings
+        run: |
+          echo "Checking for consistent line endings (LF) in source files..."          
+          files_with_crlf=$(find plugin-dev/Source/Sentry plugin-dev/Source/SentryEditor -name "*.h" -o -name "*.cpp" | xargs grep -l $'\r' 2>/dev/null || true)
+          if [ -n "$files_with_crlf" ]; then
+            echo "ERROR: Found files with CRLF line endings. Please convert all source files to use LF line endings."
+            echo "$files_with_crlf"
+            exit 1          
+          fi          
+          echo "✓ All source files have consistent LF line endings."
```

Comment:
should be doable with clang-format alone
- https://stackoverflow.com/questions/52262866/clang-format-normalized-line-endings

Reply:
you could just push the clang-formatted code automatically

Reply:
Done - `clang-format` now also checks for line endings and modifies sources in-place when needed. If any changes were made they will be pushed in the next step.